src/Apps/CollectorProfile/Routes/Saves2/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface CreateNewListValues {
+  name: string
+}
+
+export interface CreateCollectionMutationResult {
+  id: string
+}
```

Comment:
I'd avoid these types of type files and simply co-locate the type where it's primarily used. If its shared, import it from the first location

Reply:
Agree with the comment above 👆

Reply:
done

Reply:
thank you!